WalletWasabi/WabiSabi/Client/WabiSabiHttpApiClient.cs

```diff
@@ -108,6 +108,9 @@ private async Task<HttpResponseMessage> SendWithRetriesAsync(RemoteAction action
 					throw;
 				}
 			}
+
+			// Wait before the next try.
+			await Task.Delay(100, cancellationToken).ConfigureAwait(false);
```

Comment:
I think that 100m is too low. Obviously, I don't have any proof but if I were to implement this, I would use at least 250 ms and more like 500 ms. These numbers might not be great for CJ but imho 100ms is not really great value for networking purposes - imho it's too low.

Reply:
make sense, increased it.